tokio/src/runtime/blocking/pool.rs

```diff
@@ -260,6 +260,10 @@ impl Spawner {
             })
             .unwrap()
     }
+
+    pub(crate) fn is_shutdown(&self) -> bool {
```

Comment:
I wonder if we should use the `Spawner` to track the shutdown rather than the blocking pools spawner?

Reply:
Perhaps 🤷 There is already [`is_shutdown` on `Core`](https://github.com/tokio-rs/tokio/blob/master/tokio/src/runtime/thread_pool/worker.rs#L54) which is accessible through `Handle.spawner`. Is that what you have in mind?

I guess we could also add an `AtomicBool` to [`Shared`](https://github.com/tokio-rs/tokio/blob/master/tokio/src/runtime/thread_pool/worker.rs#L70) that gets set [here](https://github.com/tokio-rs/tokio/blob/master/tokio/src/runtime/thread_pool/worker.rs#L770).